data/a-a.ts

```diff
@@ -0,0 +1,32 @@
+// TODO
+const TEST_CFR = {};
+
+export default {
+  name: "A/A test",
+  description: "Use this to test audiences or to diagnose issues in the system.",
+  presets: {
+    // Specific constraints for the ui
+    max_bucket_count: 100,
+
+    // Defaults for specific experiment values
+    branches:  [
+      {
+        slug: "control",
+        is_reference_branch: true,
+        ratio: 1,
+        value: TEST_CFR,
+      },
+      {
+        slug: "treatment",
+        ratio: 1,
+        value: TEST_CFR,
+      }
+    ],
+
+    bucketing: {
+      randomization_unit: "normandy_id",
+      count: 100,
```

Comment:
At some point the bucket count will be related to `feature` and `audience`, right @tdsmith ?  So will the count always be here as part of the design and not related to those things?